common/exception/ErrorMessage.java

```diff
@@ -125,20 +125,22 @@ public static void loadConstants() {
                 new Internal(6, "Attempted to utilise a closed resource.");
         public static final Internal UNRECOGNISED_VALUE =
                 new Internal(7, "Unrecognised encoding value!");
+        public static final Internal STORAGE_PROPERTY_EXCEPTION =
+                new Internal(8, "Failed to get storage property.");
```

Comment:
It's possible RocksDB throws when we try to read a property about the SST files or the current cache status.